superset/assets/src/SqlLab/components/ScheduleQueryButton.jsx

```diff
@@ -123,12 +130,55 @@ class ScheduleQueryButton extends React.PureComponent {
   }
   renderModalBody() {
     return (
-      <Form
-        schema={getJSONSchema()}
-        uiSchema={getUISchema()}
-        onSubmit={this.onSchedule}
-        validate={getValidator()}
-      />
+      <FormGroup>
+        <Row>
+          <Col md={12}>
+            <label className="control-label" htmlFor="embed-height">
+              {t('Label')}
+            </label>
+            <FormControl
+              type="text"
+              placeholder={t('Label for your query')}
+              value={this.state.label}
+              onChange={this.onLabelChange}
+            />
+          </Col>
+        </Row>
+        <br />
+        <Row>
+          <Col md={12}>
+            <label className="control-label" htmlFor="embed-height">
+              {t('Description')}
+            </label>
+            <FormControl
+              componentClass="textarea"
+              placeholder={t('Write a description for your query')}
+              value={this.state.description}
+              onChange={this.onDescriptionChange}
+            />
+          </Col>
+        </Row>
+        <br />
```

Comment:
[nit] can we use css instead of `<br />`s to style the spacing between sections?